src/translation/translation_nl_NL.ts

```diff
@@ -2796,7 +2796,7 @@ We hebben uw kanaal gedempt en &apos;Demp mijzelf&apos; geactiveerd. Los eerst h
     <message>
         <location filename="../clientsettingsdlgbase.ui" line="986"/>
         <source>Custom Directories:</source>
-        <translation>Eigen adresboek</translation>
+        <translation>Eigen adresboeken</translation>
```

Comment:
Style question.  We've highlighted key words in Jamulus terminology in English by capitalising them (making proper nouns): have we said the translations should follow the style guide?

Reply:
(Not that the UI follows the guide yet...)